src/Ocelot.Provider.Polly/PollyPoliciesDelegatingHandler.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace Ocelot.Provider.Polly;
 
+[Obsolete("Due to new v8 policy definition in Polly 8 (use PollyResiliencePipelineDelegatingHandler)")]
```

Comment:
Better to explain to a developer to decide on method usage: old and new helpers!
So, we have to  add common method names here.

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1914/commits/c513b98ff89685fd9ad706d6c1e6b91e879cc652